Packs/PhishUp/Playbooks/PhishUp_Mail_Scanner.yml

```diff
@@ -0,0 +1,743 @@
+      version: -1
+      name: Move Mail to Spam Folder
+      description: Moves an email to a different folder.
+      script: '|||gmail-move-mail'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "19"
+    scriptarguments:
+      add-labels:
+        simple: SPAM
+      message-id:
+        simple: ${incident.labels.Email/ID}
+      remove-labels:
+        simple: INBOX
+      user-id:
+        simple: ${incident.labels.Email/Header/delivered-to}
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": -430,
+          "y": 1850
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "19":
+    id: "19"
+    taskid: b95c1bf2-b0ac-4532-877d-2fc1fb038b7e
+    type: title
+    task:
+      id: b95c1bf2-b0ac-4532-877d-2fc1fb038b7e
+      version: -1
+      name: Finish Action
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "6"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": -1390,
+          "y": 2420
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "20":
+    id: "20"
+    taskid: fc63608b-d05e-4adc-880c-8e03a7581be1
+    type: title
+    task:
+      id: fc63608b-d05e-4adc-880c-8e03a7581be1
+      version: -1
+      name: If there is a system error
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "6"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 1370,
+          "y": 880
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "21":
+    id: "21"
+    taskid: f2e9c82c-0410-4e2f-8697-c7879db780fa
+    type: title
+    task:
+      id: f2e9c82c-0410-4e2f-8697-c7879db780fa
+      version: -1
+      name: No action
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "6"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": -140,
+          "y": 1295
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+view: |-
+  {
+    "linkLabelsPosition": {},
+    "paper": {
+      "dimensions": {
+        "height": 2815,
+        "width": 3140,
+        "x": -1390,
+        "y": -130
+      }
+    }
+  }
+inputs:
+- key: Email Body Html
```

Comment:
Although it is not mandatory, it is better to name inputs without spaces or special characters, so instead `Email Body Html` --> `EmailBodyHtml`.